from aiomodels.exceptions.llm_error import LLMError

__all__ = ["LLMError"]
